### DISCUSSION TITLE
import multiple Makestyles in tss-react

### DISCUSSION THREAD

**Discussion by JohnDisilwaJD47** [garronej/tss-react#186] [Q&A]
how to import multiple makeStyles using tss-react
 const { classes } = RfpGridStyles();
  const { classes } = IntakeTableStyles();
  
  it shows redeclare block-scoped variable error

**Comment by garronej** [comment]
Hi @JohnDisilwaJD47,  

First of you should alwas prefix your hooks name by `use` it's a very important naming convention.  
Second you should never have capitalized function name (unless it's a class).  
Lastly, and this is the answer of your question, you can rename by deconstructing.  

```typescript

function MyComponent(){

    const { classes: rfpGridClasses } = useRfpGridStyles();
    const { classes: intageTableClasses } = useIntakeTableStyles();
    
    // ...

}
```  

By the way, chatGPT is very good for answering this types of questions: https://chat.openai.com/share/3006a37a-2045-4591-b1e9-bb5c7fd7f2f6
